rabbitmq/tests/compose/docker-compose.yaml

```diff
@@ -0,0 +1,8 @@
+version: '2'
```

Comment:
Can we upgrade to a newer version>

Reply:
we use v2 everywhere. I think this should work the same on v3, it's just the api version